brave/src/test/java/com/linecorp/armeria/server/brave/BraveServiceIntegrationTest.java

```diff
@@ -65,34 +67,63 @@ protected void init() {
             return HttpResponse.of(HttpStatus.NOT_FOUND);
         });
         sb.service("/foo", (ctx, req) -> HttpResponse.of(OK, MediaType.PLAIN_TEXT_UTF_8, "bar"));
-        sb.service("/extra",
+        sb.service("/async", (ctx, req) -> asyncResponse(future -> {
+            future.complete(HttpResponse.of(OK, MediaType.PLAIN_TEXT_UTF_8, "bar"));
+        }));
+
+        sb.service("/exception", (ctx, req) -> {
+            throw new IllegalStateException("not ready");
```

Comment:
I don't see how we are setting the status code 503, though.. we should put a TODO until we do?